src/main/scala/org/broadinstitute/dsde/workbench/leonardo/db/ClusterComponent.scala

```diff
@@ -61,6 +63,12 @@ trait ClusterComponent extends LeoComponent {
       }
     }
 
+    def listPending(): DBIO[Seq[Cluster]] = {
+      clusterQueryWithLabels.filter { _._1.status inSetBind ClusterStatus.pendingStatuses.map(_.toString) }.result map { recs =>
```

Comment:
nit: can we call this `monitoredStatuses` or something? pending makes me think of early in the lifecycle, whereas this covers deleting too

Reply:
👍